Applications/SystemMonitor/main.cpp

```diff
@@ -258,3 +262,57 @@ GWidget* build_file_systems_tab()
     fs_table_view->model()->update();
     return fs_widget;
 }
+
+GWidget* build_pci_devices_tab()
+{
+    auto* pci_widget = new GWidget(nullptr);
+    pci_widget->set_layout(make<GBoxLayout>(Orientation::Vertical));
+    pci_widget->layout()->set_margins({ 4, 4, 4, 4 });
+    auto* pci_table_view = new GTableView(pci_widget);
+    pci_table_view->set_size_columns_to_fit_content(true);
+
+    auto db = PCIDB::Database::open();
+
+    Vector<GJsonArrayModel::FieldSpec> pci_fields;
+    pci_fields.empend(
+        "Placement", TextAlignment::CenterLeft,
```

Comment:
Is this a good column name? I'm not terribly familiar with PCI, so this was the best i could come up with.

Reply:
I would call this "Address". That would also match the `PCI::Address` kernel object :)